app/src/main/kotlin/com/google/samples/apps/nowinandroid/MainActivity.kt

```diff
@@ -139,9 +142,10 @@ class MainActivity : ComponentActivity() {
                     androidTheme = shouldUseAndroidTheme(uiState),
                     disableDynamicTheming = shouldDisableDynamicTheming(uiState),
                 ) {
+                    val windowSize by windowSizeAsState()
```

Comment:
windowSizeAsState is now collectWindowSizeAsState

Reply:
Fixed